spk/syncthing/src/dsm-control.sh

```diff
@@ -44,34 +75,53 @@ wait_for_status ()
 
 case $1 in
     start)
-        if daemon_status; then
+        if syncthing_daemon_status; then
             echo ${DNAME} is already running
         else
             echo Starting ${DNAME} ...
-            start_daemon
+            start_syncthing_daemon
+        fi
+        if syncthing_inotify_daemon_status; then
+            echo ${DNAME_INOTIFY} is already running
+        else
+            echo Starting ${DNAME_INOTIFY} ...
+            start_syncthing_inotify_daemon
         fi
         ;;
     stop)
-        if daemon_status; then
+        if syncthing_inotify_daemon_status; then
+            echo Stopping ${DNAME_INOTIFY} ...
+            stop_syncthing_inotify_daemon
+        else
+            echo ${DNAME_INOTIFY} is not running
+        fi
+
+        if syncthing_daemon_status; then
             echo Stopping ${DNAME} ...
-            stop_daemon
+            stop_syncthing_daemon
         else
             echo ${DNAME} is not running
         fi
         ;;
     status)
-        if daemon_status; then
+        if syncthing_daemon_status; then
             echo ${DNAME} is running
-            exit 0
         else
             echo ${DNAME} is not running
             exit 1
         fi
+
+        if syncthing_inotify_daemon_status; then
+            echo ${DNAME_INOTIFY} is running
+            exit 0
+        else
+            echo ${DNAME_INOTIFY} is not running
+            exit 1
```

Comment:
Have you tested that this works with all variations? If, for example, Syncthing is _not_ running, but Inotify _is_, then I think you'll just see `Syncthing is not running`...and never see `Syncthing-Inotify is running`, because the `exit` on L111 would prevent further processing, right?
I'd also like to know that Package Center is able to correctly report the package status (where we probably want to set the status based off of Syncthing itself, not of Inotify)